bz-manage.sh

```diff
@@ -381,4 +382,137 @@ get_bz_id_summary() {
   done
 }
 
+update_doc_text() {
+  local queryurl jq bz bz_assignee doc_type doc_text gh_pr gh_pr_info gh_pr_title gh_pr_desc gh_pr_type update_bz
+  query_file=.query_file.json
+
+  # If a BZ has required_doc_text unset and GH PR set:
+  # Automatically add doc text from PR description
+  queryurl="${BASE_URL}"
+  queryurl+="&f1=flagtypes.name&f2=external_bugzilla.url&f3=ext_bz_bug_map.ext_bz_bug_id"
```

Comment:
Is there a better way to write a long string in multiple lines than doing += to the previous string?

Reply:
```suggestion
  queryurl="${BASE_URL}&f1=flagtypes.name&f2=external_bugzilla.url&f3=ext_bz_bug_map.ext_bz_bug_id"
```
?

Reply:
Maybe you mean - how to use line wrapping in bash?
```bash
queryurl="${BASE_URL}\
&f1=flagtypes.name&f2=external_bugzilla.url&f3=ext_bz_bug_map.ext_bz_bug_id\
&o1=notsubstring&o2=substring&o3=anywordssubstr"
```
then queryurl will be a very long single line string

Reply:
And if the line is very long and I want to "wrap" it?

Reply:
```bash
# foo="this is a\
very long string that is\
wrapped in a couple\
of places"
# echo "$foo"
this is avery long string that iswrapped in a coupleof places
```